### PR TITLE
Wip/hughsie/request signal

### PR DESCRIPTION
**Pull request by hughsie**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [X] Feature
- [ ] Documentation

### CODE REVIEW

**Comment by hughsie** [comment]
@superm1 other ideas welcomed if that helps.

File: plugins/test/fu-test-plugin.c [hunk 1]
```diff
@@ -229,6 +229,21 @@ fu_test_plugin_write_firmware(FuPlugin *plugin,
...
+		fwupd_request_set_id(request, FWUPD_REQUEST_ID_REMOVE_REPLUG);
+		fwupd_request_add_flag(request, FWUPD_REQUEST_FLAG_ALLOW_GENERIC_MESSAGE);
+		fwupd_request_set_message(request,
+					  "Please pretend to remove the device you cannot see or "
+					  "touch and please re-insert it. You have 5 seconds.");
+		if (!fu_device_emit_request(device, request, progress, error))
+			return FALSE;
+		g_usleep(5 * G_USEC_PER_SEC);
+	}
+
```
Comment (superm1) [inline plugins/test/fu-test-plugin.c:246]: can we add an explicit test into the installed-tests test suite for this?

Comment (superm1) [reply plugins/test/fu-test-plugin.c:246]: and if adding to installed tests, I suggest decreasing it to "1s"

Comment (hughsie) [reply plugins/test/fu-test-plugin.c]: Good idea, although, phew! Done.

**Comment by superm1** [comment]
Seeing the random CI connection failure; what do you think about moving to F39 instead now?  Are `passim-devel passim` in F39 repos?

**Comment by hughsie** [comment]
> Seeing the random CI connection failure; what do you think about moving to F39 instead now?

MSTM.

> Are passim-devel passim in F39 repos?

It is indeed -- lets see what happens in https://github.com/fwupd/fwupd/pull/6369

**Pull request merged**